samples/RaspberryPiInstructions.md

```diff
@@ -26,6 +27,8 @@ These steps have been tested on a RPi 2 and RPi 3 with Linux and Windows.
 </Project>
 ```
 
+* Add a new package source to the project with source: `https://dotnet.myget.org/F/dotnet-core/api/v3/index.json`.
```

Comment:
I think you are hitting the problem that the 2.0 SDK isn't taking over for the 1.0 SDK because its a pre-release. @livarcocc I believe is working to make 2.0 take precedence over 1.0. So, I think you are right that a lot of people will be generating the console app based on 1.0 because of this.

You can add a line that says:

Make sure you have a nuget.config file next to your csproj that includes the dotnet-core myget feed.

`<add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />`

Complete file example:
```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
  </packageSources>
</configuration>
```